### ISSUE TITLE
templates: replace glyphicons with font awesome

### ISSUE THREAD

**Issue by eamonnmag** [inveniosoftware/invenio-accounts#74]
e.g. https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/templates/invenio_accounts/login_user.html#L41 should be 

```
{{ render_field(form.email, icon="fa fa-user", autofocus=True, errormsg=False) }}
{{ render_field(form.password, icon="fa fa-lock", errormsg=False) }}
```

**Comment by lnielsen** [comment]
Thanks for the catch - will you submit a PR?

**Comment by eamonnmag** [comment]
I will :)

**Comment by lnielsen** [comment]
Fixed in f8e2fa8c0b61accd8681cbe7865e3db00a1da990
